src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -105,7 +105,7 @@ impl InitialSizeManager {
             hv.cpu_info.default_vcpus = self.resource.vcpu as i32
         }
         if self.resource.mem_mb > 0 {
-            hv.memory_info.default_memory = self.resource.mem_mb;
+            hv.memory_info.default_memory += self.resource.mem_mb;
```

Comment:
Suggest you add comments, such as the Slack discussion or golang version code reference link here.

Reply:
I have added some comments here